docs/readthedocs/source/doc/PythonAPI/Nano/tensorflow.rst

```diff
@@ -28,10 +23,18 @@ bigdl.nano.tf.optimizers
     :members: 
     :undoc-members:
 
+bigdl.nano.tf.keras.InferenceOptimizer
+---------------------------------------
+.. autoclass:: bigdl.nano.tf.keras.InferenceOptimizer
+    :members:
+    :undoc-members:
+    :exclude-members: ALL_INFERENCE_ACCELERATION_METHOD
```

Comment:
As we separate the `bigdl.nano.tf.keras.InferenceOptimizer` out of the section `bigdl.nano.tf.keras` section for better navigation purposes, should we also change the title "bigdl.nano.tf.keras" to something else?